app/actions/types.js

```diff
@@ -1,36 +1,39 @@
 export const appActions = {
-  SET_DRAWER_TAB: 'SET_DRAWER_TAB',
-  CONFIGURE_PUSH_NOTIFICATIONS: 'CONFIGURE_PUSH_NOTIFICATIONS',
-  CONNECT_TO_SOCKET_IO_SERVER: 'CONNECT_TO_SOCKET_IO_SERVER',
-  DISCONNECT_FROM_SOCKET_IO_SERVER: 'DISCONNECT_FROM_SOCKET_IO_SERVER',
-  OPEN_DRAWER: 'Navigation/OPEN_DRAWER',
-  CLOSE_DRAWER: 'Navigation/CLOSE_DRAWER',
-  HANDLE_KEYCHAIN_ERRORS: 'HANDLE_KEYCHAIN_ERRORS',
+  SET_DRAWER_TAB: "SET_DRAWER_TAB",

```

Comment:
Use single quotes instead of double quotes as all the styles the code base should be consistent.